<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TepSonic</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/mainIcon</normaloff>:/icons/mainIcon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="previousTrackButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/backward</normaloff>:/icons/backward</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playPauseButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/start</normaloff>:/icons/start</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/stop</normaloff>:/icons/stop</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextTrackButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/forward</normaloff>:/icons/forward</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loadFileButton">
        <property name="text">
         <string>Load file</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/openFile</normaloff>:/icons/openFile</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadFolderButton">
        <property name="text">
         <string>Load folder</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/openFolder</normaloff>:/icons/openFolder</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearPlaylistButton">
        <property name="text">
         <string>Clear playlist</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/clearList</normaloff>:/icons/clearList</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="trackTitleLabel">
      <property name="text">
       <string>Player is stopped</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="playerControlLayout">
      <item>
       <widget class="Phonon::SeekSlider" name="seekSlider" native="true"/>
      </item>
      <item>
       <widget class="Phonon::VolumeSlider" name="volumeSlider" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="CollectionBrowser" name="collectionBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="searchEdit"/>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="PlaylistBrowser" name="playlistBrowser">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="clearSearchButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/clearEdit</normaloff>:/icons/clearEdit</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="actionLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="actionProgress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_TepSonic">
    <property name="title">
     <string>&amp;TepSonic</string>
    </property>
    <addaction name="actionShow_Hide"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_TepSonic"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>P&amp;laylist</string>
    </property>
    <widget class="QMenu" name="menuVisible_columns">
     <property name="title">
      <string>Visible columns</string>
     </property>
     <addaction name="actionTrack"/>
     <addaction name="actionInterpret"/>
     <addaction name="actionTrackname"/>
     <addaction name="actionAlbum"/>
     <addaction name="actionGenre"/>
     <addaction name="actionYear"/>
     <addaction name="actionLength"/>
    </widget>
    <addaction name="actionAdd_file"/>
    <addaction name="actionAdd_folder"/>
    <addaction name="separator"/>
    <addaction name="actionClear_playlist"/>
    <addaction name="separator"/>
    <addaction name="actionSave_playlist"/>
    <addaction name="separator"/>
    <addaction name="menuVisible_columns"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>Pla&amp;yer</string>
    </property>
    <widget class="QMenu" name="menuRandom">
     <property name="title">
      <string>&amp;Random</string>
     </property>
     <addaction name="actionRandom_ON"/>
     <addaction name="actionRandom_OFF"/>
    </widget>
    <widget class="QMenu" name="menuRepeat">
     <property name="title">
      <string>&amp;Repeat</string>
     </property>
     <addaction name="actionRepeat_OFF"/>
     <addaction name="actionRepeat_track"/>
     <addaction name="actionRepeat_playlist"/>
    </widget>
    <addaction name="actionPrevious_track"/>
    <addaction name="actionPlay_pause"/>
    <addaction name="actionStop"/>
    <addaction name="actionNext_track"/>
    <addaction name="separator"/>
    <addaction name="menuRandom"/>
    <addaction name="menuRepeat"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_TepSonic"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionReport_a_bug"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menu_TepSonic"/>
   <addaction name="menuPlaylist"/>
   <addaction name="menuPlayer"/>
   <addaction name="menu_Settings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShow_Hide">
   <property name="text">
    <string>&amp;Show/Hide</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionLoad_folder">
   <property name="text">
    <string>Load folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuit_TepSonic">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/quit</normaloff>:/icons/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit TepSonic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd_file">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/openFile</normaloff>:/icons/openFile</iconset>
   </property>
   <property name="text">
    <string>Add &amp;file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAdd_folder">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/openFolder</normaloff>:/icons/openFolder</iconset>
   </property>
   <property name="text">
    <string>&amp;Add folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionClear_playlist">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/clearList</normaloff>:/icons/clearList</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSave_playlist">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPrevious_track">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/backward</normaloff>:/icons/backward</iconset>
   </property>
   <property name="text">
    <string>P&amp;revious track</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPlay_pause">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/start</normaloff>:/icons/start</iconset>
   </property>
   <property name="text">
    <string>&amp;Play/pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/stop</normaloff>:/icons/stop</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionNext_track">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/forward</normaloff>:/icons/forward</iconset>
   </property>
   <property name="text">
    <string>&amp;Next track</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRandom_ON">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Random O&amp;N</string>
   </property>
  </action>
  <action name="actionRandom_OFF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Random O&amp;FF</string>
   </property>
  </action>
  <action name="actionRepeat_OFF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat &amp;OFF</string>
   </property>
  </action>
  <action name="actionRepeat_track">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat &amp;track</string>
   </property>
  </action>
  <action name="actionRepeat_playlist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat &amp;playlist</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionAbout_TepSonic">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/about</normaloff>:/icons/about</iconset>
   </property>
   <property name="text">
    <string>About &amp;TepSonic</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/about</normaloff>:/icons/about</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionReport_a_bug">
   <property name="text">
    <string>&amp;Report a bug</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Track</string>
   </property>
  </action>
  <action name="actionInterpret">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interpret</string>
   </property>
  </action>
  <action name="actionTrackname">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Track name</string>
   </property>
  </action>
  <action name="actionAlbum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Album</string>
   </property>
  </action>
  <action name="actionGenre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </action>
  <action name="actionYear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </action>
  <action name="actionLength">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Length</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
  <customwidget>
   <class>PlaylistBrowser</class>
   <extends>QTreeView</extends>
   <header location="global">playlistbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>CollectionBrowser</class>
   <extends>QTreeView</extends>
   <header location="global">collectionbrowser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>previousTrackButton</tabstop>
  <tabstop>playPauseButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>nextTrackButton</tabstop>
  <tabstop>loadFileButton</tabstop>
  <tabstop>loadFolderButton</tabstop>
  <tabstop>clearPlaylistButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
